src/Sentry/SentrySdk.cs

```diff
@@ -47,36 +95,52 @@ public static class SentrySdk
         public static IDisposable Init(string dsn)
             => string.IsNullOrWhiteSpace(dsn)
                 ? DisabledHub.Instance
-                : Init(c => c.Dsn = new Dsn(dsn));
+                : Init(c => c.Dsn = new Dsn(dsn), _globalHudDefaultMode);
+
+        /// <summary>
+        /// Initializes the SDK with the specified DSN
+        /// </summary>
+        /// <remarks>
+        /// An empty string is interpreted as a disabled SDK
+        /// </remarks>
+        /// <seealso href="https://docs.sentry.io/clientdev/overview/#usage-for-end-users"/>
+        /// <param name="dsn">The dsn</param>
+        /// <param name="globalHudMode">whether to use a single (global) Hub as opposed to one per thread.</param>
+        public static IDisposable Init(string dsn, bool globalHudMode = _globalHudDefaultMode)
```

Comment:
On a second thought I believe it's best if we add this to the `SentryOptions` class and don't add any overloads here to `Init`.